cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -69,13 +70,31 @@ type NetConf struct {
 	PluginLogLevel string `json:"pluginLogLevel"`
 }
 
+// Implements encoding.TextUnmarshaler from comma-separated string.
+type IPList []net.IP
+
+func (l *IPList) UnmarshalText(data []byte) error {
+	s := bytes.Split(data, []byte(","))
+	*l = make([]net.IP, len(s))
+	for i, ipStr := range s {
+		if err := (*l)[i].UnmarshalText(ipStr); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 // K8sArgs is the valid CNI_ARGS used for Kubernetes
 type K8sArgs struct {
 	types.CommonArgs
 
 	// IP is pod's ip address
 	IP net.IP
 
+	// IPS are pod's IP addresses. It is an error if both IP and
+	// IPS are defined, and IP is not one of the addresses in IPS.
+	IPS IPList
```

Comment:
Nit: This is called `IPs` in the [containernetworking codebase](https://github.com/containernetworking/cni/blob/master/pkg/types/current/types.go#L128)

Reply:
Yes, but all the other fields in this struct are uppercase.
(I don't see why this struct needs to match the containernetworking Result struct - am I missing a connection here?)